flow/designs/asap7/eth_top/constraint.sdc

```diff
@@ -0,0 +1,4 @@
+# Set the current design
+current_design ethmac
```

Comment:
Why is setting the current design necessary? This should already be done in the script before the SDC is loaded

Reply:
@rovinski yes its part of Makefile. I will clean up this as well